docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -31,6 +31,420 @@ For more information about versions, build numbers, and Office Online Server, se
+||[addGeometricShape(geometricShapeType: Excel.GeometricShapeType, left: number, top: number, width: number, height: number)](/javascript/api/excel/excel.shapecollection#addgeometricshape-geometricshapetype--left--top--width--height-)|Adds a geometric shape to worksheet. Returns a Shape object that represents the new shape.|
+||[addGroup(values: Array<string \| Shape>)](/javascript/api/excel/excel.shapecollection#addgroup-values-)|Group a subset of shapes in a worksheet. Returns a Shape object that represents the new group of shapes.|
+||[addImage(base64ImageString: string)](/javascript/api/excel/excel.shapecollection#addimage-base64imagestring-)|Creates an image from a base64 string and adds it to worksheet. Returns the Shape object that represents the new Image.|
+||[addSVG(xmlImageString: string)](/javascript/api/excel/excel.shapecollection#addsvg-xmlimagestring-)|Creates an SVG from a XML string and adds it to worksheet. Returns a Shape object that represents the new Image.|
+||[addTextBox(text?: string)](/javascript/api/excel/excel.shapecollection#addtextbox-text-)|Adds a textbox to worksheet by telling it's text content. Returns a Shape object that represents the new text box.|
+||[getCount()](/javascript/api/excel/excel.shapecollection#getcount--)|Returns the number of shapes in the worksheet. Read-only.|
+||[getItem(shapeId: string)](/javascript/api/excel/excel.shapecollection#getitem-shapeid-)|Returns a shape identified by the shape id. Read-only.|
+||[load(option?: OfficeExtension.LoadOption)](/javascript/api/excel/excel.shapecollection#load-option-)|Queues up a command to load the specified properties of the object. You must call "context.sync()" before reading the properties.|
+||[items](/javascript/api/excel/excel.shapecollection#items)|Gets the loaded child items in this collection.|
+|[ShapeDeactivatedEventArgs](/javascript/api/excel/excel.shapedeactivatedeventargs)|[shapeId](/javascript/api/excel/excel.shapedeactivatedeventargs#shapeid)|Gets the id of the shape that is deactivated.|
+||[type](/javascript/api/excel/excel.shapedeactivatedeventargs#type)|Gets the type of the event. See Excel.EventType for details.|
+||[worksheetId](/javascript/api/excel/excel.shapedeactivatedeventargs#worksheetid)|Gets the id of the worksheet in which the shape is deactivated.|
+|[ShapeFill](/javascript/api/excel/excel.shapefill)|[clear()](/javascript/api/excel/excel.shapefill#
```

Comment:
for detail -> for details.

In places like this where we're saying "See [foo] for details." is it easily doable to make [foo] a hyperlink?  (If so, this change should be implemented anywhere a description references another (API Reference) article.)

Reply:
This is a bigger change we could do across the whole reference API set. We could do it for only this table, but I'm not sure there's a ton of value.